packages/components/src/toggle-group-control/stories/index.tsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { formatLowercase, formatUppercase } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ToggleGroupControl,
+	ToggleGroupControlOption,
+	ToggleGroupControlOptionIcon,
+} from '../index';
+import type {
+	ToggleGroupControlOptionProps,
+	ToggleGroupControlOptionIconProps,
+	ToggleGroupControlProps,
+} from '../types';
+
+const meta: ComponentMeta< typeof ToggleGroupControl > = {
+	component: ToggleGroupControl,
+	title: 'Components (Experimental)/ToggleGroupControl',
+	subcomponents: { ToggleGroupControlOption, ToggleGroupControlOptionIcon },
+	argTypes: {
+		// TODO: Should this component be non-polymorphic?
+		as: { control: { type: null } },
```

Comment:
@ciampo Thoughts? I don't think it makes sense for this to be polymorphic.

Reply:
Makes sense — this component has quite strong semantics, afterall